server/src/FileChangeListener.ts

```diff
@@ -3,9 +3,14 @@ import { DidChangeWatchedFilesParams, Emitter, Event } from 'vscode-languageserv
 import { DbtProject } from './DbtProject';
 import { DbtRepository } from './DbtRepository';
 import { ManifestParser } from './manifest/ManifestParser';
+import { debounce } from './utils/Utils';
 
 export class FileChangeListener {
+  private static PACKAGES_UPDATE_DEBOUNCE_TIMEOUT = 1000;
```

Comment:
Can you check how many times event will be called with this value and find value to have only one event call (or just as little as possible times)?

Reply:
Added logs. According to them debounce timeout starts after last update of tracked folder.